test/check-application

```diff
@@ -2122,6 +2122,19 @@ class TestApplication(testlib.MachineCase):
         self.execute(auth, f"podman inspect --format '{{{{.Id}}}}' {container_name_new}").strip()
         self.waitContainerRow(container_name_new)
 
+    def testMultipleContainers(self):
+        self.login()
+
+        # Create 31 containers
+        for i in range(31):
+            self.execute(True, f"podman run -dt --name container{i} quay.io/libpod/busybox:latest")
+
+        self.waitContainerRow("container30")
+
+        # Generic cleanup takes too long and timeouts, so remove these container manually one by one
+        for i in range(31):
+            self.execute(True, f"podman rm -f container{i}")
```

Comment:
`m.execute()` has a `timeout=` argument. I wonder if it's faster to do

    podman ps -aq --filter 'name=^container' | xargs podman rm -f

If not, then keep it like that.

Reply:
> podman ps -aq --filter 'name=^container' | xargs podman rm -f

that would also timeout. I could specify `timeout=` in `execute()` but would be kinda tricky to find the correct number that will not keep it stuck for too long if one container is stuck, yet is long enough for 30 containers.